library/alloc/src/task.rs

```diff
@@ -33,6 +33,7 @@ pub trait Wake {
     }
 }
 
+#[allow(rustc::ineffective_unstable_trait_impl)]
```

Comment:
Can you add a short comment here explaining why this is ok?

Something like
```suggestion
// While this impl is automatically stable, it only applies to types implementing `Wake`,
// which is unstable and not implemented for any stable type.
#[allow(rustc::ineffective_unstable_trait_impl)]
```